packages/gdpr-tool/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@commercetools/gdpr-tool",
+    "version": "1.0.0-development",
+    "description": "Export and delete all data related to a single customer",
```

Comment:
Is that the only use case for this library? 🤔

Reply:
Yup.. Maybe we could even fit this inside of another package in the monorepo?